packages/gafl-webapp-service/src/pages/order-complete/licence-details/licence-details.njk

```diff
@@ -39,21 +21,44 @@
             <h1 class="govuk-heading-xl">{{ title }}</h1>
 
             <p class="govuk-body-l">
-                {{ mssgs.licence_details_body }}
+                {{ mssgs.licence_details_body_1 }}
+            </p>
+
+            <p class="govuk-body-l">
+                {{ mssgs.licence_details_body_2 }}
             </p>
         </section>
 
         <section class="pagebreak">
 
             <img src="/public/images/EA-logo-black.png" class="print-area ea-logo">
 
-            <span class="govuk-caption-xl"><span class="print-area">{{ mssgs.licence_num }}</span></span>
+            {% for licence in data.licences %}
 
-            <h2 class="govuk-heading-l">{{ data.permission.referenceNumber }}</h2>
+                {% set licenseDetailsSummaryRows = [
+                    { key: { text: mssgs.licence_summary_name }, value: { text: licence.licenceHolder } },
+                    { key: { text: mssgs.identification }, value: { text: licence.obfuscatedDob } },
+                    { key: { text: mssgs.licence_summary_type }, value: { text: licence.type } },
+                    { key: { text: mssgs.licence_summary_length }, value: { text: licence.length } },
+                    { key: { text: mssgs.starts }, value: { text: licence.start } },
+                    { key: { text: mssgs.ends }, value: { text: licence.end } }
+                ]
+                %}
 
-            {{ govukSummaryList({
-                rows: licenseDetailsSummaryRows
-            }) }}
+                {% if licence.disabled %}
+                    {% set licenseDetailsSummaryRows = (licenseDetailsSummaryRows.push({ key: { text: mssgs.licence_summary_disability_concession }, value: { text: mssgs.yes } }), licenseDetailsSummaryRows) %}
+                {% endif %}
+                {% if licence.ageConcession %}
+                    {% set licenseDetailsSummaryRows = (licenseDetailsSummaryRows.push({ key: { text: mssgs.age_concession }, value: { text: licence.ageConcession } }), licenseDetailsSummaryRows) %}
+                {% endif %}
+
+                <h2 class="govuk-heading-l">{{ licence.referenceNumber }}</h2>
+
+                {{ govukSummaryList({
+                    rows: licenseDetailsSummaryRows
+                }) }}
```

Comment:
Did you create the ticket to refactor this in the way we've done for other summary pages?

Reply:
yeah its: https://eaflood.atlassian.net/browse/IWTF-3153